Sources/SpeziQuestionnaire/QuestionnaireView.swift

```diff
@@ -51,7 +51,7 @@ public struct QuestionnaireView: View {
                 isPresented: $isPresented,
                 questionnaireResponse: { response in
                     await questionnaireResponse?(response)
-                    await questionnaireDataSource.add(response)
+                    await questionnaireDataSource.add(response) // TODO: remove that thingy
```

Comment:
There is still a TODO in here. Happy to discuss what we should do here in a few minutes.